bin/pypeit_ql_keck_mosfire

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+
+
+"""
+This script runs PypeIt
+"""
+
+from pypeit.scripts import ql_keck_mosfire
```

Comment:
If this script is to be supported going forth, we need tests and docs.

Reply:
Yes, I understand this is the beta version.